### PR TITLE
Added filter for disable responsive condition for theme

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Added filter to disable/enable display condition for core.

### Screenshots

- NA

### Types of changes

- NA

### How has this been tested?

- NA

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/extensions/condition-block/index.js [hunk 1]
```diff
@@ -223,8 +224,7 @@ const AdvancedControlsBlock = createHigherOrderComponent( ( BlockEdit ) => {
 						>
 							{ __( 'Filter to disable responsive/display condition. ' ) }
 						</ExternalLink>
-						<hr className="uagb-editor__separator" />
-						{ UserResponsiveConditionOptions( props ) }
+						{ '1' === enableResponsiveConditionsForCoreBlocks && UserResponsiveConditionOptions( props ) }
```
Comment (RajkiranBagal) [inline src/blocks/extensions/condition-block/index.js:227]: Why we have checked '1'? '1' === true gives false.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/extensions/condition-block/index.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/extensions/condition-block/index.js:227]: By default `enableResponsiveConditionsForCoreBlocks` is true which return in js as 1 so we are showing responsive condition only when it is 1 else hide it. @RajkiranBagal

Comment (RajkiranBagal) [reply src/blocks/extensions/condition-block/index.js:227]: Have you confirmed this case? @AbhijitNage123

Comment (AbhijitNage123) [reply src/blocks/extensions/condition-block/index.js:227]: @RajkiranBagal Yes confirm when it set to false it return empty and for true it set to 1

**Pull request merged**